packages/components/src/InputNumber/InputNumber.mdx

```diff
@@ -110,49 +110,6 @@ Input number is used in forms that accept numbers as an answer.
   </Text>
 </Playground>
 
-## Using onValidate
-
-If you need to capture the error message and render it outside of the component.
```

Comment:
I wonder if we should include a link here to the `InputText` validation docs?

Reply:
You's right. I'll send them to `InputValidation` tho just because the example there is mostly focused on it